### PR TITLE
gMax 1.5+ Configurations

### PR DESCRIPTION
**Pull request by gordo3di**
gMax 1.5+ Single and Dual extruder configurations. We spent a full week testing our settings and are happy with the results. These were tested in Cura 3.1 as its the latest stable release and we set up the extruders, quality, variants, etc... based on the recommended templates.

Note:
We left in material_print_temperature in our start code knowing this issue #3074, #3061, #3032, #1296 has been resolved in 3.2

Thanks everyone

### CODE REVIEW

File: resources/definitions/desktop.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[.ShellClassInfo]
```
Comment (ChrisTerBeke) [inline resources/definitions/desktop.ini]: This seems to be a file from another tool, not needed by Cura. Please remove or ignore it from Git.

File: resources/extruders/desktop.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[.ShellClassInfo]
```
Comment (ChrisTerBeke) [inline resources/extruders/desktop.ini]: See above.

File: resources/meshes/desktop.ini [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[.ShellClassInfo]
```
Comment (ChrisTerBeke) [inline resources/meshes/desktop.ini]: See above.

File: resources/definitions/desktop.ini (hunk 1 above)
Comment (ChrisTerBeke) [reply resources/definitions/desktop.ini]: There's more of these, please remove all.

Comment (gordo3di) [reply resources/definitions/desktop.ini]: Fixed. Forgot i had hidden files off on this computer.

**Comment by gordo3di** [comment]
Cleared out the 6 desktop.ini files. Check is still failing?

**Comment by gordo3di** [comment]
@fieldOfView Thanks.

**Pull request merged**